cmd/manager/main.go

```diff
@@ -189,3 +207,21 @@ func serveCRMetrics(cfg *rest.Config) error {
 	}
 	return nil
 }
+
+// checkIfRequiredFilesExist checks for the existence of required files and binaries before starting WMCO
+// sample error message: errors encountered with required files: stat /payload/hybrid-overlay.exe: no such file or directory,
+// stat /payload/wmcb.exe: no such file or directory
+func checkIfRequiredFilesExist(requiredFiles []string) error {
+	var errors []string
+	// Iterating through file paths and checking if they are present
+	for _, file := range requiredFiles {
+		if _, err := os.Stat(file); err != nil {
+			errors = append(errors, err.Error())
```

Comment:
Could you also add context as for which file is experiencing the error?
`errors = append(errors, sprintf("could not stat %s: %s"), file, err.Error())`